loadtest/control/actions.go

```diff
@@ -29,6 +29,7 @@ type UserActionResponse struct {
 // user.User. It returns a UserActionResponse.
 type UserAction func(user.User) UserActionResponse
 
+// SignUp is the action that a user does when signing up to the server.
```

Comment:
I would try to avoid the "is the action" part and just go ahead and explain what it does like the other methods. Something like "SignUp signs up a user to the server". Yes, it's slightly odd, but that's what Go docs suggest. :/